File: src/Worker.java
public class Worker extends Person implements ICommunication{
    private CV cv;
    private Industry industry;
    private String description;

    public CV getCv() {
        return cv;
    }

    public void setCv(CV cv) {
        this.cv = cv;
    }

    public Industry getIndustry() {
        return industry;
    }

    public void setIndustry(Industry industry) {
        this.industry = industry;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    @Override
    public void sendMessage(String sender, String receiver, String message) {

    }
}

File: PHP/src/Worker.php
<?php

class Worker extends Person
{
    private int $id;
    public static array $WORKER_LIST = array();
    private Industry $industry;
    private bool $isFreeToWork;
    private CV $cv;

    /**
     * @param int $id
     * @param Industry $industry
     */
    public function __construct(int $id, String $name, Industry $industry)
    {
        $this->id = $id;
        $this->industry = $industry;
        $this->name = $name;
        $this->registerInSystem();
    }

    /**
     * @return Industry
     */
    public function getIndustry(): Industry
    {
        return $this->industry;
    }

    /**
     * @param Industry $industry
     */
    public function setIndustry(Industry $industry): void
    {
        $this->industry = $industry;
    }

    /**
     * @return bool
     */
    public function isFreeToWork(): bool
    {
        return $this->isFreeToWork;
    }

    /**
     * @param bool $isFreeToWork
     */
    public function setIsFreeToWork(bool $isFreeToWork): void
    {
        $this->isFreeToWork = $isFreeToWork;
    }

    public function __toString(): string
    {
        return "{$this->name} {$this->id} working in {$this->industry->name}";
    }

    public function registerInSystem(): void
    {
        array_push(self::$WORKER_LIST, $this);
    }

    public function updateCV(CV $cv):void
    {
        $this->cv = $cv;
    }
}